packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -510,6 +514,11 @@ export interface DbIndexEntry {
   arrayValue: Uint8Array;
   /** The encoded directional value for equality and inequality filters. */
   directionalValue: Uint8Array;
+  /**
+   * The document key this entry points to. This entry is encoded by an ordered
+   * encoder to match the key order of the index.
+   */
+  orderedDocumentKey: Uint8Array;
```

Comment:
Is it true that if the order was always ASC, then using `EncodedResourcePath` below would have sufficed? in order words: is the only reason we need this is because of DESC indexes?

Reply:
FWIW, I changed the `documentKey` entry to no longer use `EncodedResourcePath` since we no longer rely on any ordering.